Content.Server/GameObjects/Components/WiresComponent.cs

```diff
@@ -479,8 +479,11 @@ bool IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
             return true;
         }
 
-        void IExamine.Examine(FormattedMessage message)
+        void IExamine.Examine(FormattedMessage message, bool inDetailsRange)
         {
+            if (!inDetailsRange)
```

Comment:
You can normally see this on the sprite, can't you? Should it be a detail?

Reply:
The size of the sprite varies, usually its a few pixels.

Possibly noticeable but possibly not, I made dice examinable from infinite range since you can see the side on their sprite and they're fairly small, so I guess for consistency I'll revert this part.